### PR TITLE
Add algorithm for Newton's Law of Gravitation

### PR DESCRIPTION
**Pull request by ZeroDayOwl**
### Describe your change:
I implemented an algorithm to find out the values of either Force of attraction, distance or magnitude of any one of the masses of a two-body gravitational system using Newton's Universal Law of Gravitation, provided the values of the other three parameters are non-zero numbers.

- [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
- [x] All new Python files are placed inside an existing directory.
- [x] All filenames are in all lowercase characters with no spaces or dashes.
- [x] All functions and variable names follow Python naming conventions.
- [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
- [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by ZeroDayOwl** [comment]
Hey @cclauss, sorry for pinging. Could you please take a look at this PR?

File: physics/newtons_law_of_gravitation.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    >>> gravitational_law(force=-847938e12, mass_1=674, mass_2=0, distance=9374)
+    Traceback (most recent call last):
+        ...
+    ValueError: Gravitational force can not be negative
+    """
+
+    product_of_mass = mass_1 * mass_2
+
+    if (force, mass_1, mass_2, distance).count(0) != 1:
+        raise ValueError("Only one argument must be 0")
```
Comment (cclauss) [inline physics/newtons_law_of_gravitation.py:75]: OPTIONAL:
```suggestion
        raise ValueError("One and only one argument must be 0")
```

Comment (cclauss) [reply physics/newtons_law_of_gravitation.py:75]: LGTM.  Let me know if you want this change or not and then we can merge.

**Review by cclauss — APPROVED** [review]
Rock & Roll -- Thanks for doing this!

**Comment by ZeroDayOwl** [comment]
Thanks for merging!

**Pull request merged**